app/views/docs/oauth-providers/google.md

```diff
@@ -0,0 +1,355 @@
+# Google provider
+
+Appwrite allows you to authenticate users using their Google account through the Google OAuth2 provider. OAuth authentication is a great way to reduce friction for your users and increase user conversion by simplifying the signup process. You can learn more about Appwrite's other OAuth2 providers [here](https://appwrite.io/docs/client/account?sdk=web-default#accountCreateOAuth2Session).
+
+## Enabling the Google provider
+
+Before you can use Google to authenticate users, you need to enable the provider in your Appwrite console.
+
+1. Navigate to your Appwrite project
+2. Navigate to **Auth** > **Settings**
+3. Find and open the OAuth provider
+4. In the **Google OAuth2 Settings** modal, use the toggle to enable the provider
+
+Don't close this modal, we'll need an OAuth client ID to complete this form.
+
+## Creating a Google OAuth2.0 client
+
+To use Google OAuth with Appwrite, you need to create an OAuth 2.0 client from your google cloud console. You can do this by following the [Setting up OAuth 2.0](https://support.google.com/googleapi/answer/6158849) guide from Google.
+
+- When prompted to provide an **Application type**, select **web application**, irrespective of your target platform. This is because the communication occurs between Appwrite (a Web App) and the OAuth2 provider.
```

Comment:
```suggestion
- When prompted to provide an **Application type**, select **web application**, irrespective of your target platform. This is because the communication occurs between Appwrite (a web app) and the OAuth2 provider on behalf of your web or mobile app.
```